config/locales/en/publisher_information/publisher_updates.yml

```diff
@@ -0,0 +1,55 @@
+en:
+  publisher_information:
+    publisher_updates:
+      title: What’s new in Whitehall Publisher
+      summary: |
+        Summary of updates to Whitehall Publisher, content design guidance, or the design of GOV.UK.
+      last_updated: Last updated 5 September 2022
+      intro_section:
+        heading: Moving to the Design System
+        body_govspeak: |
+          Whitehall Publisher will be moving from the current styling to use the styling and components of the [GOV.UK Design System](https://design-system.service.gov.uk/).
+          This will make Whitehall Publisher more accessible, more user friendly and give you to have a consistent user experience throughout your publishing journey on GOV.UK.
+
+          [Read more about the changes on the content community Basecamp](https://3.basecamp.com/4322319/buckets/15005645/messages/5287245970).
+      sections:
+        - heading: Recent changes
+          body_govspeak: |
```

Comment:
Rather than one big block of govspeak should we make the updates to be an array.

e.g:

```
updates:
  - heading: Reusing previous withdrawal dates and public explanations
    tag: Improved
    body_govspeak: |
      When re-withdrawing content, you can choose to reuse a previous withdrawal date and public explanation in certain circumstances.

      [Read the guidance about when you can reuse previous withdrawal dates and explanations](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/unpublishing-and-archiving).
  - heading: Shareable document preview links
    tag: New
    body_govspeak: |
      You can share document preview links for draft content with people who do not have access to Whitehall Publisher.

      [Read the guidance about where and how to use document preview links](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#preview-content).
```

Reply:
I did suggest this but @johnpauldickie said it would be better to just have it as one block in case they want to do something different for one update.

Reply:
Do we have an idea of what freedom is hoped for? Seems unlikely one wouldn't have a header and tag can be optional and then it's just as free in the body govspeak.

Be helpful if you record these decisions in commit messages too.

Reply:
This has been updated to match suggestion